src/components/data-exploration/cell-sets-tool/CellSetsTool.jsx

```diff
@@ -32,6 +33,19 @@ const { Text } = Typography;
 
 const { TabPane } = Tabs;
 
+const generateIsFilteredCellIndicator = (geneExpressions) => {
+  const someGeneExpressionObj = _.find(geneExpressions, () => true);
```

Comment:
What exactly are you doing here? `geneExpressions` is an array and my guess is you want an arbitrary element, right? Why not simply use `[0]`?